.github/workflows/auto-publish.this_should_be_yml

```diff
@@ -0,0 +1,19 @@
+# .github/workflows/pr-push.yml
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
```

Comment:
previously, this was only done on the branch "publish_wd". Are you sure you want main? (it seems so but want to be sure)

Reply:
Yeah, I think doing it simple is fine. The repo is dormant anyway, so if we wake up, this can be changed if we want.